### PR TITLE
cherry pick qos driver code change to F5 driver stable/queens

### PR DESCRIPTION
**Pull request by zhang-shengping**
(cherry picked from commit af49f798107564c8937a59c7865c8ede4d5fbed3)

Conflicts:
	f5lbaasdriver/v2/bigip/service_builder.py

### CODE REVIEW

File: f5lbaasdriver/v2/bigip/service_builder.py [hunk 1]
```diff
@@ -203,6 +210,18 @@ def _get_extended_member(self, context, member):
...
+    @log_helpers.log_method_call
+    def _get_extended_qos(self, context, loadbalancer):
+        try:
+            sess = keystone.get_session()
+            client_sess = client.Client(session=sess)
+            project = client_sess.projects.get(project=loadbalancer.tenant_id)
+            return project.qos
+
+        except Exception as e:
+            LOG.error('Exception: Get keystone project: %s', e.message)
```
Comment (jikui) [inline f5lbaasdriver/v2/bigip/service_builder.py]: should we change the log level from error to warning or some lower levels?

Comment (zhang-shengping) [reply f5lbaasdriver/v2/bigip/service_builder.py]: > should we change the log level from error to warning or some lower levels?

OK,  I will change this bit in this branch :)

**Review by jikui — APPROVED** [review]

File: f5lbaasdriver/v2/bigip/service_builder.py [hunk 2]
```diff
@@ -18,13 +18,15 @@
 import json
 
 from oslo_config import cfg
+from neutron_lbaas.common import keystone
```
Comment (Niklaus-xie) [inline f5lbaasdriver/v2/bigip/service_builder.py:20]: 这里看来要放到上面一行

Comment (zhang-shengping) [reply f5lbaasdriver/v2/bigip/service_builder.py:20]: Yes, you are right

**Pull request merged**